explore-analyze/query-filter/languages/esql-cross-clusters.md

```diff
@@ -14,11 +14,6 @@ products:
 # Use ES|QL across clusters [esql-cross-clusters]
 
```

Comment:
````suggestion
```{applies_to}
stack: preview 9.0, ga 9.1
```
````
This is how version-related changes are conveyed in the new docs as they must cover multiple versions at once. They will render as badges in the output.